acceptance/examples/rule_data.rego

```diff
@@ -0,0 +1,14 @@
+package main
+
+deny[result] {
+    not data.rule_data__configuration__
+
+    result := "No custom rule data"
+}
+
+deny[result] {
+    object.get(data.rule_data__configuration__, "custom", "") != "data1"
```

Comment:
Just to clarify, given the scenario that uses this file, it'll never be the case that both `custom` and `other` are both set, right?

Reply:
Well… I realized later that we only invoke conftest once with all sources, I was almost certain that we would invoke it once per source